Corona-Warn-App/src/main/res/values/strings.xml

```diff
@@ -193,223 +193,223 @@
-    <string name="risk_details_behavior_body_cough_sneeze">Husten und Niesen Sie in Ihre Armbeuge</string>
-    <string name="risk_details_behavior_increased_body_1">- Ihren Hausartzt/Ihre Hausärtztin</string>
-    <string name="risk_details_behavior_increased_body_2">- Den Kassenärtzlichen Notdienst (Tel: 116117)</string>
-    <string name="risk_details_behavior_increased_body_3">- Ihr Gesundheitsamt</string>
-    <string name="risk_details_headline_infection_risk">Infektionsrisiko</string>
-    <string name="risk_details_body_infection_risk">So wird Ihr Risiko ermittelt</string>
+    <string name="risk_details_headline_behavior">Behave correctly</string>
+    <string name="risk_details_behavior_body_stay_home">Stay at home</string>
+    <string name="risk_details_behavior_body_avoid_contact">Avoid social contact</string>
+    <string name="risk_details_behavior_body_contact_doctor">If you have any questions about symptoms, testing options and further secretion measures, please contact:</string>
+    <string name="risk_details_behavior_body_wash_hands">Wash your hands</string>
+    <string name="risk_details_behavior_body_wear_mask">Wear a protective mask</string>
+    <string name="risk_details_behavior_body_stay_away">Keep a distance of 2 meters to other people</string>
+    <string name="risk_details_behavior_body_cough_sneeze">Cough and sneeze in the crook of your arm</string>
+    <string name="risk_details_behavior_increased_body_1">- Your general practitioner</string>
+    <string name="risk_details_behavior_increased_body_2">- The emergency medical service (Phone: 116117)</string>
+    <string name="risk_details_behavior_increased_body_3">- Your health office</string>
+    <string name="risk_details_headline_infection_risk">Risk of infection</string>
+    <string name="risk_details_body_infection_risk">How your risk gets determined</string>
     <string name="risk_details_button_update">@string/risk_card_button_update</string>
     <string name="risk_details_button_enable_tracing">@string/risk_card_button_enable_tracing</string>
     <!-- TODO check if needed -->
-    <string name="main_government_headline">Gesundheitsamt anrufen</string>
+    <string name="main_government_headline">Call the health department</string>
 
     <!-- ####################################
               Onboarding
     ###################################### -->
     <string name="onboarding_title">Onboarding</string>
     <string name="onboarding_image_description">IS MISSING</string>
-    <string name="onboarding_button_finish">Zustimmen</string>
-    <string name="onboarding_button_disable">Nicht aktivieren</string>
-    <string name="onboarding_button_next">Weiter</string>
-    <string name="onboarding_button_start">Los Geht\'s</string>
-    <string name="onboarding_button_back_description">Zurück</string>
+    <string name="onboarding_button_finish">Agree</string>
+    <string name="onboarding_button_disable">Not enable</string>
```

Comment:
May be "Disagree"?

Reply:
I would translate it word by word. In the German version the string is "Nciht aktivieren", which definitely not means "Disagree"

Reply:
Then can we change this also to "Don't enable?"

Reply:
Yes. I'll just change it